bundles/org.openhab.binding.meteoalerte/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -200,6 +304,7 @@
 	<channel-type id="comment">
 		<item-type>String</item-type>
 		<label>Commentaire</label>
+		<category>text</category>
```

Comment:
I believe this is the implicit value for a String item.

Reply:
No, this had no category

Reply:
What I mean is that if you do not set any category, OH will determine one based on the widget type. For a channel type linked to an item of type String, a Text widget will be used and this will lead to category "Text" being used. At least in sitemap UIs.

Reply:
Yes, but not in Main UI

Reply:
That would mean a category should be set on every channel type. A lot are certainly missing.